client/src/Main.elm

```diff
@@ -614,24 +595,16 @@ addBackboneView model =
         [ el [ Element.Region.heading 1, Font.size 50 ] <| Element.text "Add new Backbone"
         , Input.text []
             { onChange = ChangeBackboneNameToAdd
-            , text = model.nameBackboneToAdd
+            , text = Maybe.withDefault "" <| Maybe.map .name model.backboneToAdd
             , label = Input.labelLeft [] <| Element.text "Name:"
             , placeholder = Nothing
             }
         , Input.text []
             { onChange = ChangeBackboneMpgNumberToAdd
-            , text = model.mpgbNumberBackboneToAdd
+            , text = String.fromInt <| Maybe.withDefault 0 <| Maybe.map .mPGBNumber model.backboneToAdd
```

Comment:
```suggestion
            , text =  Maybe.withDefault "" <| Maybe.map (String.fromInt << .mPGBNumber) model.backboneToAdd
```

Reply:
Done!
I've also done the same for the level 0 table.